### DISCUSSION TITLE
GAJOE/CRYSOL - Chi2 values less than 1

### DISCUSSION THREAD

**Discussion by I-G-E** [biosaxs-com/atsas-community#1962] [Mixtures and flexible systems]
Hi,

I have run GAJOE and CRYSOL for a number of systems and have a question about the Chi2 values. I know that a good fit would be a Chi2 value of 1. However what about a Chi2 less than 1? For some of my datasets, I have a Chi2 value of 0.99 or 0.94. Based upon slide 19 in this presentation www.embl-hamburg.de/biosaxs/courses/embo2018/slides/franke-statistics.pdf , it seems that for a SAXS dataset containing 500 datapoints, a Chi2 of 0.99 or 0.94 would be acceptable as a good fit, and would not be classified as 'overfitting'. But would a Chi2 of 0.94 be better than a Chi2 of 1 or is a Chi2 of 1 still the ideal, best solution?

Thank you.

**Comment by franke-biosaxs** [comment]
Numerically, smaller may be considered "better". Many people see it this way. Statistically speaking, in either case the Hypothesis that "the model fits the data" can not be rejected with any degree of significance. To me, both are equally valid models for the SAXS data at hand.

 If you have to pick one over the other, maybe consider other methods to evaluate and rank your model as well?

**Reply by I-G-E** [reply]
Thank you for your explanation and suggestion. I've heard some people say that anything under 1 would be considered over-fitting. Would you disagree?

**Reply by franke-biosaxs** [reply]
Depends. On your error estimates and the chi-square value obtained. If your errors are ok, a value of 0.99 is not overfitting. If the errors are correct, would expect a value of "around 1.0" and get a value of 0.5, that is overfitting. If your errors are wrong, e.g. by re-using buffers, you might get a value of 0.5 - which can't be interpreted without resolving the depends that were introduced by re-using buffers.

As long as the chi-square is within the bulk of the expected chi-square distribution, the results should be fine. This can, depending on the number of points, be a range of say 0.9-1.1 (less points), or 0.99-1.01 (a lot of points).
